examples/remove_widget_from_formDockWidget_example.py

```diff
@@ -0,0 +1,84 @@
+import sys
+
+from PySide2 import QtWidgets
+
+from eqt.ui import UIFormWidget
+
+
+class MainUI(QtWidgets.QMainWindow):
+    def __init__(self, parent=None):
+        QtWidgets.QMainWindow.__init__(self, parent)
+        dock = UIFormWidget.FormDockWidget(parent=self)
+        dock.setWindowTitle('Example remove widget')
+
+        # add widget 1 as QLineEdit
+        qlabel = QtWidgets.QLabel(dock)
+        qlabel.setText("Widget 1: ")
+        qwidget = QtWidgets.QLineEdit(dock)
+        qwidget.setClearButtonEnabled(True)
+        dock.addWidget(qwidget, qlabel, 'Widget 1')
+
+        # add widget 2 as QLineEdit
+        qlabel = QtWidgets.QLabel(dock)
+        qlabel.setText("Widget 2: ")
+        qwidget = QtWidgets.QLineEdit(dock)
+        qwidget.setClearButtonEnabled(True)
+        dock.addWidget(qwidget, qlabel, 'Widget 2')
+
+        # add widget 3 as QLineEdit
+        qlabel = QtWidgets.QLabel(dock)
+        qlabel.setText("Widget 3: ")
+        qwidget = QtWidgets.QLineEdit(dock)
+        qwidget.setClearButtonEnabled(True)
+        dock.addWidget(qwidget, qlabel, 'Widget 3')
+
+        # add input as QComboBox
+        dock.widget().addSpanningWidget(QtWidgets.QLabel("Pick the widget you want to remove: "),
```

Comment:
I notice you have to do this because the FormDockWidget class doesn't have an 'addSpanningWidget' method. 
I know this isn't the point of your PR but please can you add it, because then it will make your form creation code easier to reuse between the two examples.
Link to the FormDockWidget: https://github.com/TomographicImaging/eqt/blob/abf3fc015ca8b552ceffe3bb933cef7aaf17b240/eqt/ui/UIFormWidget.py#L314

Reply:
Then it will mean you can call dock.addSpanningWidget rather than having to call dock.widget().addSpanningWidget

Reply:
I agree, but in the future we should accept the role from QFormLayout ItemRole, see https://github.com/TomographicImaging/eqt/issues/92